ui/js/models/perf/alerts.js

```diff
@@ -217,6 +217,23 @@ treeherder.factory('PhAlerts', [
             return _.find(phAlertSummaryStatusMap, { id: this.status }).text;
         };
 
+        AlertSummary.prototype.notesDidntChange = (this.notes === this.original_notes);
+        AlertSummary.prototype.saveNotes = function () {
+            const alertSummary = this;
+            return $http.put(`${SERVICE_DOMAIN}/api/performance/alertsummary/${this.id}/`,
```

Comment:
Could you switch this to using the helpers in:
https://github.com/mozilla/treeherder/blob/master/ui/helpers/urlHelper.js